examples/py-hypertrace/configs/example-srtmnet.json

```diff
@@ -42,8 +42,8 @@
                         "aerosol_template_path": "../../data/aerosol_template.json",
                         "earth_sun_distance_file": "../../data/earth_sun_distance.txt",
                         "irradiance_file": "../../examples/20151026_SantaMonica/data/prism_optimized_irr.dat",
-                        "emulator_aux_file": "./sRTMnet_v100/sRTMnet_v100_aux.npz",
-                        "emulator_file": "./sRTMnet_v100/sRTMnet_v100",
+                        "emulator_aux_file": "./sRTMnet_v120/sRTMnet_v120_aux.npz",
+                        "emulator_file": "./sRTMnet_v120/sRTMnet_v120",
```

Comment:
And this isn't an H5. Should this also be an H5?

Reply:
Yes, good catch! Fixed.